### PR TITLE
get data from hatcheries

### PR DESCRIPTION
**Pull request by AustinRS016**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 venv/
 .DS_Store
-.idea
\ No newline at end of file
+.idea
+
```
Comment (ghost) [inline .gitignore:4]: nit: new line

File: python/getHatcheryData.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+load_dotenv()
+
+client = Socrata("data.wa.gov",
+                 os.getenv('APP_KEY'),
+                 username= os.getenv('APP_USERNAME'),
+                 password= os.getenv('APP_PASSWORD'))
+
+# Limit defaults to 1000, not sure what to put for 'no-limit' dataset is ~450,000 rows
+#  use limit=1 for testing purposes and limit=10000000 for development
+results = client.get("9q4e-xhag", limit=1000)
```
Comment (ghost) [inline python/getHatcheryData.py]: https://dev.socrata.com/docs/queries/select.html

We probably should just use this if the API provides it

File: python/config.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+              'SOOS CREEK HATCHERY': ('WA_HATCHERY_DATA',), 'SPEELYAI HATCHERY': ('WA_HATCHERY_DATA',), 'TOKUL CREEK HATCHERY': ('WA_HATCHERY_DATA',),
+              'TUCANNON HATCHERY': ('WA_HATCHERY_DATA',), 'TUMWATER FALLS HATCHERY': ('WA_HATCHERY_DATA',), 'VOIGHTS CREEK HATCHERY': ('WA_HATCHERY_DATA',),
+              'WASHOUGAL HATCHERY': ('WA_HATCHERY_DATA',), 'FOSTER RD TRAP': ('WA_HATCHERY_DATA',), 'KALAMA FALLS HATCHERY': ('WA_HATCHERY_DATA',),
+              'LK ABERDEEN HATCHERY': ('WA_HATCHERY_DATA',), 'MERWIN DAM FCF': ('WA_HATCHERY_DATA',), 'MODROW TRAP': ('WA_HATCHERY_DATA',),
+              'SOLDUC HATCHERY': ('WA_HATCHERY_DATA',), 'SUNSET FALLS FCF': ('WA_HATCHERY_DATA',), 'WALLACE R HATCHERY': ('WA_HATCHERY_DATA',),
+              'WASHOUGAL RIVER FISH WEIR': ('WA_HATCHERY_DATA',), 'WHITEHORSE POND': ('WA_HATCHERY_DATA',)}
+
+
+class Hatcheries(Enum):
+    WA_HATCHERY_DATA = auto()
```
Comment (ghost) [inline python/config.py:5]: nice job

File: python/config.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+              'PRIEST RAPIDS HATCHERY': ('WA_HATCHERY_DATA',), 'REITER PONDS': ('WA_HATCHERY_DATA',), 'RINGOLD SPRINGS HATCHERY': ('WA_HATCHERY_DATA',),
+              'SAMISH HATCHERY': ('WA_HATCHERY_DATA',), 'SKAMANIA HATCHERY': ('WA_HATCHERY_DATA',), 'SKOOKUMCHUCK HATCHERY': ('WA_HATCHERY_DATA',),
+              'SOOS CREEK HATCHERY': ('WA_HATCHERY_DATA',), 'SPEELYAI HATCHERY': ('WA_HATCHERY_DATA',), 'TOKUL CREEK HATCHERY': ('WA_HATCHERY_DATA',),
+              'TUCANNON HATCHERY': ('WA_HATCHERY_DATA',), 'TUMWATER FALLS HATCHERY': ('WA_HATCHERY_DATA',), 'VOIGHTS CREEK HATCHERY': ('WA_HATCHERY_DATA',),
+              'WASHOUGAL HATCHERY': ('WA_HATCHERY_DATA',), 'FOSTER RD TRAP': ('WA_HATCHERY_DATA',), 'KALAMA FALLS HATCHERY': ('WA_HATCHERY_DATA',),
+              'LK ABERDEEN HATCHERY': ('WA_HATCHERY_DATA',), 'MERWIN DAM FCF': ('WA_HATCHERY_DATA',), 'MODROW TRAP': ('WA_HATCHERY_DATA',),
+              'SOLDUC HATCHERY': ('WA_HATCHERY_DATA',), 'SUNSET FALLS FCF': ('WA_HATCHERY_DATA',), 'WALLACE R HATCHERY': ('WA_HATCHERY_DATA',),
+              'WASHOUGAL RIVER FISH WEIR': ('WA_HATCHERY_DATA',), 'WHITEHORSE POND': ('WA_HATCHERY_DATA',)}
+
```
Comment (ghost) [inline python/config.py]: nit: new line

File: python/getHatcheryData.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+from sodapy import Socrata
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
```
Comment (ghost) [inline python/getHatcheryData.py:8]: nit: extra line again

File: python/getHatcheryData.py [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+                            OR species='Steelhead'
+                            OR species='Chum' 
+                            OR species='Pink' 
+                            OR species='Sockeye' 
+                            )"""
+                         )
+    return results
+
+
+prin
```
Comment (ghost) [inline python/getHatcheryData.py]: nit: i think this is left overs

**Review by ghost — APPROVED** [review]
LGTM with a few nits and some leftovers

File: python/config.py (hunk 3 above)
Comment (ghost) [reply python/config.py:5]: Oh wait, you aren't using your enum in the data

File: python/config.py [hunk 7]
```diff
@@ -0,0 +1,23 @@
+from enum import Enum, auto
+
+hathceries = {'BAKER LAKE HATCHERY': ('WA_HATCHERY_DATA',), 'BEAVER CREEK HATCHERY': ('WA_HATCHERY_DATA',), 'BINGHAM CREEK HATCHERY': ('WA_HATCHERY_DATA',),
```
Comment (ghost) [inline python/config.py]: hathceries = {'BAKER LAKE HATCHERY': (Hatcheries.WA_HATCHERY_DATA,), 

Might have to define class Hatcheries above this

**Review by ghost — CHANGES_REQUESTED** [review]
You should use the enum in the data

**Comment by AustinRS016** [comment]
My autoformatter has been adding these new lines.. Once you get a linter set up I'll turn it off.

**Pull request merged**